runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldTableRow.java

```diff
@@ -65,14 +71,24 @@
 		FLAG_UK = new ImageIcon(ImageUtil.getResourceStreamFromClass(WorldHopperPlugin.class, "flag_uk.png"));
 		FLAG_US = new ImageIcon(ImageUtil.getResourceStreamFromClass(WorldHopperPlugin.class, "flag_us.png"));
 		FLAG_GER = new ImageIcon(ImageUtil.getResourceStreamFromClass(WorldHopperPlugin.class, "flag_ger.png"));
+
+		final BufferedImage starOnImage = ImageUtil.getResourceStreamFromClass(IconTextField.class, "star.png");
+
+		ICON_FAVORITE_ON = new ImageIcon(starOnImage);
+		ICON_FAVORITE_ON_HOVER = new ImageIcon(ImageUtil.grayscaleOffset(starOnImage, -100));

```

Comment:
Generally speaking, `alphaOffset` should be used in place of `grayscaleOffset` for hovered images.